corehq/apps/translations/app_translations.py

```diff
@@ -514,11 +517,8 @@ def get_module_detail_graph_rows(langs, detail, list_or_detail):
 
 
 def get_form_question_rows(langs, form):
-    if form.form_type == 'shadow_form':
```

Comment:
It's intentional that this check is inside `get_form_question_rows`, rather than have the caller responsible for checking the form type.